libtenzir/fbs/table_slice.fbs

```diff
@@ -10,6 +10,9 @@ table v2 {
   /// A timestamp assigned on import, representing nanoseconds since the UNIX
   /// epoch.
   import_time: long;
+
+  /// TODO
+  internal: bool;
```

Comment:
This is neat, I really like this way of doing it. Much better than checking whether the schema name begins with `tenzir.`.